app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -86,24 +85,37 @@
             app:layout_constraintStart_toStartOf="@+id/admin_auth_heading_textview"
             app:layout_constraintTop_toBottomOf="@id/admin_auth_heading_textview" />
 
-          <org.oppia.android.app.profile.ProfileInputView
+          <com.google.android.material.textfield.TextInputLayout
             android:id="@+id/admin_auth_input_pin"
+            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="168dp"
             android:layout_marginTop="48dp"
             android:layout_marginEnd="168dp"
             android:layout_marginBottom="4dp"
-            android:addTextChangedListener="@{viewModel.inputPinTextWatcher}"
-            android:imeOptions="actionDone"
-            app:inputLength="5"
-            app:isPasswordInput="true"
-            app:label="@string/admin_auth_pin"
+            app:boxCornerRadiusBottomEnd="4dp"
```

Comment:
the radius for the border. 
We can create a style for it which can be useable for every other place.

Reply:
done